test/e2e/manage-type-vcluster.sh

```diff
@@ -26,16 +26,16 @@ set -e # exit on error
 : -------------------------------------------------------------------------
 : Wait for the running component of ControlPlane cp2 to be ready/completed
 :
-kubectl --context kind-kubeflex -n cp2-system wait --for=jsonpath='{.status.availableReplicas}'=1 statefulset/vcluster
-kubectl --context kind-kubeflex -n cp2-system wait --for=condition=Complete job/update-cluster-info --timeout=60s
+kubectl --context kind-kubeflex -n cp2-system wait --for=jsonpath='{.status.availableReplicas}'=1 statefulset/vcluster --timeout=120s
+kubectl --context kind-kubeflex -n cp2-system wait --for=condition=Complete job/update-cluster-info --timeout=120s
 
 :
 : -------------------------------------------------------------------------
 : Create a Deployment in ControlPlane cp2, then wait for the Deployment
 : to become available, with default timeout which is 30 seconds
 :
-kubectl --context cp2 create deployment my-nginx --image nginx
-kubectl --context cp2 wait --for=condition=Available deploy/my-nginx
+kubectl --context cp2 create deployment my-nginx --image nginx 
```

Comment:
I suspect that the added space here is unintended.

Reply:
I added a --timeout=120s  in all `kubectl wait` to account for delays in images download that may fail the test. There might have been some change in spaces.